howto/install-appliance/aws.md

```diff
@@ -0,0 +1,136 @@
+
+![Start the Launch Instance Wizard|690x451](https://assets.ubuntu.com/v1/17073a3d-install_appliance_launch-wizard.png)
+
+### 2. Select the AMI
+
+To select the Amazon Machine Image (AMI), type "Anbox Cloud" in the search field of the **Application and OS Images** section.
+
+![Search for the Anbox Cloud Appliance AMI|690x451](../../images/install_appliance_search-ami.png)
+
+Choose either the Arm variant or the x86 variant and click **Select**.
+
+![Select the Amazon Machine Image (AMI)|690x451](../../images/install_appliance_select-ami.png)
+
+You will be presented with the pricing information. Click **Continue** to confirm.
+
+### 3. Choose an instance type
+
+AWS offers various instance types. The Anbox Cloud Appliance images are supported for a subset of the available instance types only.
+
+In the **Instance type** section, select the instance type that is most suitable for what you're planning to do. For example, if you just want to try out the Anbox Cloud Appliance, an instance type with GPU support and limited CPU and memory is sufficient. See the [Requirements](https://discourse.ubuntu.com/t/installation-requirements/17734#appliance) for the minimum hardware requirements.
+
+![Choose an instance type|690x451](../../images/install_appliance_instance-type.png)
+
+In this example, we picked `g4dn.2xlarge`, which provides 8 vCPUs, 32 GB of memory and a single NVIDIA Tesla T4 GPU.
+
+### 4. Select a key pair
+
+In the **Key pair (login)** section, choose an existing key pair or create one if you don't have one yet. Make sure to save the private key in a secure location.
+
+![Choose or create a key pair|690x451](../../images/install_appliance_key-pair.png)
+
+### 5. Configure the network
+
+You do not need to customise any of the settings in the **Network settings** section, but you can fine-tune things. For example, you might want to put the instance onto a different VPC or subnet.
+
+To allow external access, several ports in the security group attached to the AWS instance must be open. The AMI already comes with the required configuration, so you don't need to do any changes. For reference, all required ports are documented [here](https://discourse.ubuntu.com/t/requirements/17734).
+
+![Configure the security group|690x451](../../images/install_appliance_security-group.png)
+
+### 6. Add storage
+
+The Anbox Cloud instance requires sufficient storage to work correctly. The root disk should have at minimum 50 GB and for best performance, you should create an additional EBS volume of at least 50 GB. Anbox Cloud uses the additional volume exclusively to store all of its data, including containers. Using a separate volume isolates it from the operating system, which increases performance. If no additional EBS volume is added, the Anbox Cloud Appliance automatically creates an image on the root disk, which is used to store any data. However, this is not recommended.
```

Comment:
Can we somewhat emphasize this point? We have seen several people deploying with just a root disk of 50GB and then had issues making things work

Reply:
I broke it up into a list now, which should make it more visible than just a block of text.

The alternative is to require the second disk instead of only recommend it, if we want to be even more clear?